utils/provided-goi-analysis-functions.R

```diff
@@ -6,34 +6,35 @@
 #
 # source(file.path("utils", "provided-goi-analysis-functions.R"))
 
-colnames_to_gene_symbols <- function(normalized_sce_matrix,
+colnames_to_plotting_symbols <- function(normalized_sce_matrix,
                                     goi_list,
-                                    ensembl_id_column,
-                                    gene_symbol_column) {
+                                    sce_rownames_column,
+                                    optional_plotting_column) {
```

Comment:
```suggestion
                                    plotting_column) {
```
I'm a little torn on switching up the name, but for this specific function it isn't optional, so I think I would remove that part.